examples/ansible/ansibleRun-inline-advanced.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: ansible.crossplane.io/v1alpha1
+kind: AnsibleRun
+metadata:
+  annotations:
+    ansible.crossplane.io/runPolicy: ObserveAndDelete
+  name: gcpbucket
+spec:
+  forProvider:
+    # PlaybookSet default to using a remote source - like playbookSet-remote.yaml.
```

Comment:
To avoid the possible confusion to users, I think the comments here may need to be revised as the notion of `PlaybookSet` is something we previously used and has been replaced by `AnsibleRun`. Same applies to `ansibleRun-inline.yaml`.

Another thing I just realized is that the current `AnsibleRun` CR uses some spec fields that do not align with the design such as `source` and `module`, see the sample snippets at https://github.com/multicloudlab/crossplane-provider-ansible/blob/main/docs/design.md#inline where we actually do not have `source` and `module`, but use `playbookInline` instead.